spec/formats/news_story_spec.rb

```diff
@@ -29,7 +29,7 @@ def and_i_fill_in_the_form_fields
     fill_in "revision[summary]", with: "A great summary"
 
     document = Document.first
-    base_path = document.document_type.path_prefix + "/a-great-title"
+    base_path = document.current_edition.document_type.path_prefix + "/a-great-title"
```

Comment:
You could probably switch this to be `Edition.first` and not need to look up document.